addons/l10n_ar_withholding/__manifest__.py

```diff
@@ -5,7 +5,6 @@
     'version': "1.0",
     'description': """Allows to register withholdings during the payment of an invoice.""",
     'author': 'ADHOC SA',
-    'countries': ['ar'],
```

Comment:
This change does not fix anything, since this module is not auto-installable.
There are several cases like this in your branch.

Reply:
okay, I will fix it.
so I only remove it if it's auto-installed and depends on any localization.

Reply:
done